### DISCUSSION TITLE
My custom theme is not showing up in the admin portal

### DISCUSSION THREAD

**Discussion by MikeAlhayek** [OrchardCMS/OrchardCore#11037] [Q&A] [answered]
I created a new theme called `TheDefaultAdminTheme` which inherits from the standard `TheAdmin` theme. The only difference is a minor changes to the view "content fields, bag Part, and Title part...". I followed the info [here ](https://docs.orchardcore.net/en/dev/docs/guides/create-admin-theme/)

The generated dll is called `TheDefaultAdminTheme.dll` the package on nuget.org is called [CrestApps.OC.Themes.TheDefaultAdminTheme](https://www.nuget.org/packages/CrestApps.OC.Themes.TheDefaultAdminTheme). When I install the project in the web/host project using NuGet.org feed, it does not show up on the admin UI and I can't use it in my app. But, if add a reference to the theme project using the actual project, it works and it shows up as expected.

Here is the dependencies list 

![image](https://user-images.githubusercontent.com/24724371/150043601-44e0b9c6-1c6d-40fd-b9dc-b0b91db82e1b.png)

The manifest file looks like this

```
using OrchardCore.DisplayManagement.Manifest;
using OrchardCore.Modules.Manifest;

[assembly: Theme(
    Name = "The Default Admin Theme (an extension of TheAdmin)",
    Id = "TheDefaultAdminTheme",
    Author = "Mike Alhayek",
    Website = "https://crestapps.com",
    Version = "1.1.0",
    Description = "More compact views for TheAdmin theme",
    Tags = new[] { ManifestConstants.AdminTag },
    BaseTheme = "TheAdmin"
)]

```
What am I missing here? How can I make the theme accessible after its installed using nuget.org?

**Comment by Psichorex** [comment]
Did you add your theme as a dependency in the asp.net project?

**Reply by MikeAlhayek** [reply]
I did. found the problem. look at my answer

**Comment by MikeAlhayek** [comment, marked as answer]
It turned out that the nuget package id must be the same as the project name for OC to see it. Its strange to me that the nuget package id just be the same as the dll for OC to see it. This is a working package https://www.nuget.org/packages/TheDefaultAdminTheme/

**Reply by hishamco** [reply]
Exactly the Theme should be same as project name

**Reply by jtkech** [reply]
More precisely equal to the assembly name which by default is equal to the project name
